resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/LocalityPlacementStrategySuite.scala

```diff
@@ -57,7 +62,6 @@ class LocalityPlacementStrategySuite extends SparkFunSuite {
     // goal is to create enough requests for localized containers (so there should be many
     // tasks on several hosts that have no allocated containers).
 
-    val resource = Resource.newInstance(8 * 1024, 4)
```

Comment:
https://github.com/apache/spark/pull/28566#issuecomment-629961204

Reply:
ah, yeah. Seems it's used in the branch-3.0. I will fix.

Reply:
should be fixed now. thanks for pointing out quickly.